molo/core/wagtail_hooks.py

```diff
@@ -62,6 +65,6 @@ def show_main_language_only(parent_page, pages, request):
 def register_import_menu_item():
     return MenuItem(
         _('Import content'),
-        urlresolvers.reverse('import-from-ucd'),
+        urlresolvers.reverse('import-from-git'),
```

Comment:
I'm stealing the current seemingly unused `Import content` menu item at the moment, but happy to make room in the UI for both of these import options.